doc/conf.py

```diff
@@ -84,6 +84,7 @@ def __getattr__(cls, name):
 MOCK_MODULES = [
     "mlir_quantum",
     "mlir_quantum.runtime",
+    "mlir_quantum.runtime.np_to_memref",
```

Comment:
Is it not jnp?

Reply:
No. There are two modules. One by the MLIR runtime library which is just `np_to_memref`. The one I created extends this module to allow similar functionality for JAX arrays.